debugger/src/main/scala/org.apache.daffodil.tdml/TDML.scala

```diff
@@ -278,6 +280,8 @@ object TDML {
                 .asInstanceOf[JAXBElement[DocumentPartType]]
                 .getValue()
                 .getValue()
+                .replaceFirst("^\\s++", "")
+                .replaceFirst("\\s++$", "")
```

Comment:
Use `.trim()`.

Reply:
.trim() only removes a whitespace and it does not remove multiple whitespaces before and after. We also used .strip() function but it is only available post Java11 and make it noncompatible with Java8